src/dcf_manifest_generator.py

```diff
@@ -39,9 +39,11 @@ def generate_dcf(self):
         acl ="['*']" if not self.submission.get("controlledAccess", False) else f"['{self.submission.get('dbGaPID')}']"
         authz = "['/open']" if not self.submission.get("controlledAccess", False) else f"['/programs/{self.submission.get('dbGaPID')}']"
         url =  f's3://{self.config[PROD_BUCKET_CONFIG_NAME]}/{self.submission[DATA_COMMON_NAME]}/{self.submission.get("studyID")}/'
+        
         for r in file_nodes:
+            node_id = r[NODE_ID] if r[NODE_ID].startswith("dg.4DFC/") else "dg.4DFC/" + r[NODE_ID],
```

Comment:
Could you extract the prefix into a constant? Thanks!

Reply:
I convert lots string values to constants in new branch for CRDCDH-1355 and will be ready for your review soon.

Reply:
updated.